bundle/manifests/yugabyte-platform-operator.clusterserviceversion.yaml

```
@@ -54,68 +87,87 @@ metadata:
           }
         }
       ]
-    capabilities: Deep Insights
-    categories: "Database"
-    description: Operator for Yugabyte Platform makes it easy to deploy and upgrade Yugabyte platform on Openshift environments.
-    containerImage: registry.connect.redhat.com/yugabytedb/yugabyte-platform-operator:1.0.0
-    createdAt: 2020-07-16T12:00:00Z
-    support: YugabyteDB
+    capabilities: Seamless Upgrades
+    categories: Database
     certified: "true"
+    containerImage: registry.connect.redhat.com/yugabytedb/yugabyte-platform-operator:latest
+    description: Operator for Yugabyte Platform makes it easy to deploy and upgrade Yugabyte platform on Openshift environments.
+    operators.operatorframework.io/builder: operator-sdk-v1.0.0
+    operators.operatorframework.io/project_layout: helm.sdk.operatorframework.io/v1
     repository: https://github.com/yugabyte/yugabyte-platform-operator
-  name: yugabyte-platform-operator.v1.0.0
+    support: YugabyteDB
+  name: yugabyte-platform-operator.v0.0.1
   namespace: placeholder
 spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
-    - description: deploy and manage Yugabyte Platform as a kubernetes native custom resource viz. yugawares.charts.helm.k8s.io
-      kind: Yugaware
+    - kind: YBPlatform
+      name: ybplatforms.yugabyte.com
       version: v1alpha1
-      displayName: yugawares.charts.helm.k8s.io
-      name: yugawares.charts.helm.k8s.io
+      description: Deploy and manage Yugabyte Platform as a kubernetes native custom resource .
+      displayName: Yugabyte Platform
       resources:
-        - kind: ServiceAccount
-          version: v1
-        - kind: deployments
-          version: v1
-        - kind: pods
-          version: v1
-      specDescriptors: []
-      statusDescriptors: []
-  description: |
+      - kind: ServiceAccount
+        name: ""
+        version: v1
+      - kind: deployments
+        name: ""
+        version: v1
+      - kind: pods
+        name: ""
+        version: v1
+      - kind: servicemonitors
+        name: ""
+        version: v1
+      - kind: clusterrolebindings
+        name: ""
+        version: v1
+      - kind: statefulsets
+        name: ""
+        version: v1
+      - kind: poddisruptionbudgets
+        name: ""
+        version: v1beta1
+      - kind: persistentvolumeclaims
+        name: ""
+        version: v1
+      - kind: clusterroles
+        name: ""
+        version: v1
+      - kind: replicasets
+        name: ""
+        version: v1
+      - kind: secrets
+        name: ""
+        version: v1
+      - kind: services
+        name: ""
+        version: v1
+      - kind: configmaps
+        name: ""
+        version: v1
+      - kind: Yugaware
+        name: ""
+        version: v1alpha1
```

```suggestion
      - kind: ServiceAccount
        name: ""
        version: v1
      - kind: StatefulSet
        name: ""
        version: v1
      - kind: PodDisruptionBudget
        name: ""
        version: v1beta1
      - kind: PersistentVolumeClaim
        name: ""
        version: v1
      - kind: Secret
        name: ""
        version: v1
      - kind: Service
        name: ""
        version: v1
      - kind: ConfigMap
        name: ""
        version: v1
```
This is a list for users to understand what all things our operator manages. These resources should be more that enough. https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md#owned-crds